bin/api2html.js

```diff
@@ -69,6 +70,10 @@ if (program.args.length === 0) {
     options.language_tabs = [];
     options.sample = !program.raw;
 
+    if (program.groups) {
+        options.tagGroups = JSON.parse(fs.readFileSync(program.groups, 'utf8'));
```

Comment:
Please wrap this in a try...catch block and fail gracefully when the path is not correct/existing